stdlib/2and3/hmac.pyi

```diff
@@ -11,7 +11,10 @@ _Hash = Any
 
 digest_size: None
 
-if sys.version_info >= (3, 4):
+if sys.version_info >= (3, 8):
+    def new(key: _B, msg: Optional[_B] = ...,
+            digestmod: Union[str, Callable[[], _Hash], ModuleType] = ...) -> HMAC: ...
```

Comment:
It's a bit of a weird situation, but since you must provide the `digestmod` argument in 3.8 or you get an exception I don't think it would be correct to say that it has a default value. I'd prefer to see two overloads: One where all three arguments must be given and one similar to `def new(key, *, digestmod)`. This would still allow to leave out `msg` when `digestmod` is given as a keyword argument, but requires `digestmod` in all cases.